src/coreclr/src/dlls/mscoree/coreclr/CMakeLists.txt

```diff
@@ -180,51 +181,45 @@ if(FEATURE_SINGLE_FILE_DIAGNOSTICS)
 endif(FEATURE_SINGLE_FILE_DIAGNOSTICS)
 
 if(CLR_CMAKE_TARGET_WIN32)
-    if (NOT CMAKE_GENERATOR MATCHES "Visual Studio .*")
```

Comment:
We can remove this since we aren't building DacTableGen with the C# language support in CMake (that's the part that was Visual Studio only).

Additionally, we're removing the NMake support and replacing it with Ninja. To my knowledge, the only usage of NMake was to generate a compile_commands.json file for jit-format, and Ninja can do that as well.